invenio_jobs/templates/semantic-ui/invenio_jobs/system/jobs/jobs-details.html

```diff
@@ -14,7 +14,12 @@
     class="ten wide mobile twelve wide tablet thirteen wide computer fourteen wide widescreen column"
   >
     <div class="ui container fluid pl-10 pr-10 pt-10 pb-10">
-      {{ go_back() }}
+      <div class="rel-mb-2">
+        <a href="/administration/jobs" class="ui mini button labeled icon rel-mr-1">
```

Comment:
could you check if we can use the new invenio_url_for function to resolve the url from the view name? it would be better not to hardcode the urls in the code